app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -241,7 +241,7 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         val maybePocketFragment = (fragmentManager.findFragmentByTag(PocketVideoFragment.FRAGMENT_TAG) as PocketVideoFragment?)?.let {
             if (it.isVisible) it else null
         }
-        TelemetryIntegration.INSTANCE.saveControllerInformation(applicationContext, event.deviceId)
+        TelemetryIntegration.INSTANCE.saveRemoteControlInformation(applicationContext, event)
```

Comment:
One more nit! Name this `keyEvent` to be clear - because we use a lot of events so this might as well be explicit.